packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +135,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? new ClampingScrollPhysics(),
```

Comment:
Specificially, this is the class I'd like to pass: https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/animation/home.dart#L371